miryoku/mapping/42/eyelash_corne.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 Manna Harbour
+// https://github.com/manna-harbour/miryoku
+
+#if !defined (MIRYOKU_LAYOUTMAPPING_EYELASH_CORNE)
+
+#define XXX &none
+
+#define MIRYOKU_LAYOUTMAPPING_EYELASH_CORNE( \
+     K00, K01, K02, K03, K04,                     K05, K06, K07, K08, K09, \
+     K10, K11, K12, K13, K14,                     K15, K16, K17, K18, K19, \
+     K20, K21, K22, K23, K24,                     K25, K26, K27, K28, K29, \
+     N30, N31, K32, K33, K34,                     K35, K36, K37, N38, N39 \
+) \
+XXX  K00  K01  K02  K03  K04            XXX       K05  K06  K07  K08  K09  XXX \
+XXX  K10  K11  K12  K13  K14       XXX  XXX  XXX  K15  K16  K17  K18  K19  XXX \
+XXX  K20  K21  K22  K23  K24  XXX       XXX       K25  K26  K27  K28  K29  XXX \
+               K32  K33  K34                      K35  K36  K37
+
+#define MIRYOKU_KLUDGE_TOPROWCOMBOS_LEFTPINKIE 2 3
+#define MIRYOKU_KLUDGE_TOPROWCOMBOS_LEFTINNERINDEX 3 4
+#define MIRYOKU_KLUDGE_TOPROWCOMBOS_RIGHTINNERINDEX 7 8
+#define MIRYOKU_KLUDGE_TOPROWCOMBOS_RIGHTPINKIE 8 9
+
+#define MIRYOKU_KLUDGE_BOTTOMROWCOMBOS_LEFTPINKIE 26 27
+#define MIRYOKU_KLUDGE_BOTTOMROWCOMBOS_LEFTINNERINDEX 27 28
+#define MIRYOKU_KLUDGE_BOTTOMROWCOMBOS_RIGHTINNERINDEX 31 32
+#define MIRYOKU_KLUDGE_BOTTOMROWCOMBOS_RIGHTPINKIE 32 33
+
+#define MIRYOKU_KLUDGE_THUMBCOMBOS_LEFT 37 38
+#define MIRYOKU_KLUDGE_THUMBCOMBOS_RIGHT 39 40
+
```

Comment:
The combo key positions need updating, but since they are optional they can all just be deleted.